### DISCUSSION TITLE
Automatically switch from curve2 to curve1 after 5 seconds

### DISCUSSION THREAD

**Discussion by cellardoorends** [Rem0o/FanControl.Releases#1510] [Q&A] [answered]
Hi. When Fan Control launches after windows boot, is it possible to make it apply curve2 that a made with 60% fan speed and after 3/5/10 seconds for example apply curve1 for the rest of session? 
EVGA 3080 FTW3 has a problem with fan3 when it constantly changes rpm back and forth untill you ramp it to a certain speed and then it stops. But the speed to make this issue is too high, around 1200 rpm for me. So what i do is i open Fan Control and applying curve2 manually which makes fan3 spin at 1200-1300 rpm, then after few seconds i apply curve1 with 500 rpm and problem goes away. But i need to do this every time i boot PC.
My english is not ideal, sorry.

**Comment by Rem0o** [comment, marked as answer]
There is no such feature exactly. There is a start% where if your fan it stopped and it is about to start, it will go to let say 60% until it spins properly then it will go to whatever % you set it to.  But that behavior only happens if the fan is initially stopped.

**Reply by cellardoorends** [reply]
This maybe be exactly what i need. The fan is stopped only until the Fan Control launches. https://i.imgur.com/x9LiGH2.png is this it ? or i need another type of curve for this?

**Reply by cellardoorends** [reply]
I set it to 60% and the problem is gone. Thats crazy. Thank you!
